server/src/main.rs

```diff
@@ -169,75 +372,32 @@ async fn main() -> Result<(), Error> {
 
     #[cfg(not(debug_assertions))]
     let backend = Backend::dynamo().await;
+
     #[cfg(debug_assertions)]
-    let backend = if std::env::var_os("USE_DYNAMODB").is_some() {
-        Backend::dynamo().await
-    } else {
+    let backend = {
         use rand::prelude::SliceRandom;
-        use serde::Deserialize;
-        use std::time::Duration;
-
-        #[cfg(debug_assertions)]
-        #[derive(Deserialize)]
-        struct LiveAskQuestion {
-            likes: usize,
-            text: String,
-            hidden: bool,
-            answered: bool,
-            #[serde(rename = "createTimeUnix")]
-            created: usize,
-        }
 
-        let mut state = Local::default();
-        let seed: Vec<LiveAskQuestion> = serde_json::from_str(SEED).unwrap();
-        let seed_e = "00000000000000000000000000";
-        let seed_e = Ulid::from_string(seed_e).unwrap();
-        state.events.insert(seed_e, String::from("secret"));
-        state.questions_by_eid.insert(seed_e, Vec::new());
-        let mut state = Backend::Local(Arc::new(Mutex::new(state)));
-        let mut qs = Vec::new();
-        for q in seed {
-            let qid = ulid::Ulid::new();
-            state
-                .ask(
```

Comment:
The old version would specifically call `ask` to re-use the logic from there, which you've opted not to do in your version. Any particular reason for that? I think it'd simplify the logic especially for the DynamoDB case.

Reply:
In the old logic there was a mix of injecting data to the state directly and doing so via the backend.'s api Keeping it would require to pack the state back  to the `Backend::Local`  which was the case previously.

Reply:
Notice how we needed to inject  `answered`, `hidden`, `when` to the state manually after calling the backend's public api. I am pretty sure we did so because the api did not allow to also set those to values we need when creating a new question

Reply:
Yep, though I still think that saves you a bunch of annoying setup. Essentially, you use `ask` to add all the questions themselves, and then you directly reach into the backend to manipulate the metadata (vote counts, hidden/answered).